### PR TITLE
SA1401 - do not report this diagnostic for const fields

### LINKED ISSUE #384: Consider disabling SA1401 FieldsMustBePrivate for constants
Currently SA1401 is reported for `const` fields. Considering these fields cannot be encapsulated in a property without changing their semantics, it seems SA1401 should be disabled for this case.

### PR DESCRIPTION
**Pull request by macpak**
Fixes #384.

### CODE REVIEW

File: StyleCop.Analyzers/StyleCop.Analyzers.Test/MaintainabilityRules/SA1401UnitTests.cs [hunk 1]
```diff
@@ -95,7 +95,18 @@ public class SA1401UnitTests : CodeFixVerifier
             await VerifyCSharpDiagnosticAsync(testCode, EmptyDiagnosticResults, CancellationToken.None);
         }
 
-        protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
+		[TestMethod]
```
Comment (pdelvo) [inline StyleCop.Analyzers/StyleCop.Analyzers.Test/MaintainabilityRules/SA1401UnitTests.cs]: The indentation is off here

**Comment by macpak** [comment]
Seems like github problem, because when I look at the code in VS it looks ok.

**Comment by pdelvo** [comment]
You are using Tabs instead of spaces here this is why it looks good in your editor

**Comment by macpak** [comment]
Seems like update of VS changed it. I will fix it thanks !

**Pull request merged**